beacon/beacon.go

```diff
@@ -714,7 +736,10 @@ func (pd *ProtocolDriver) listenEpochs(ctx context.Context) {
 			layer = epoch.Add(1).FirstLayer()
 
 			pd.setProposalTimeForNextEpoch()
-			pd.logger.With().Info("processing epoch", current, epoch)
+			pd.logger.Info("processing epoch",
+				zap.Uint32("layer_id", current.Uint32()),
+				zap.Uint32("epoch_id", epoch.Uint32()),
+			)
```

Comment:
These are not "IDs". How about just layer and epoch?
```suggestion
			pd.logger.Info("processing epoch",
				zap.Uint32("layer", current.Uint32()),
				zap.Uint32("epoch", epoch.Uint32()),
			)
```

Reply:
The reason I named them like this is because that's how they are logged before the change. I can adapt the key to the suggestion 🤷